docs/package.json

```diff
@@ -47,7 +47,7 @@
     "rehype-slug": "^4.0.1",
     "remark-code-import": "^0.3.0",
     "remark-frontmatter": "^3.0.0",
-    "remark-gfm": "^1.0.0",
+    "remark-gfm": "^3.0.1",
```

Comment:
Should add the `generatePropsTables` to a build script so we don't forget to run it?

Reply:
Good catch.

As mentioned in the PR description, I removed the script in this commit https://github.com/aws-amplify/amplify-ui/commit/9bd0431cd49cb0f10d2b6a3a6e5c55c43a2ac66f for some build failures, but I can add it back only for manual script, no build time run 🤔